README.md

```diff
@@ -88,6 +88,24 @@ The Central Ledger has many options that can be configured through environment v
 | CLEDG\_AMOUNT__PRECISION | Numeric value used to determine precision recorded for transfer amounts on this ledger. | 10 |
 | CLEDG\_AMOUNT__SCALE | Numeric value used to determine scale recorded for transfer amounts on this ledger. | 2 |
 
+### Kafka Position Event Type Action Topic Map
+
+In some cases, you might want to publish position type messages onto a customized topic name that
+diverges from the defaults.
+
+You can configure the customized topic names in the config. Each position action key
+refers to position messages with associated actions.
```

Comment:
Please add a note, that ONLY "POSITION.PREPARE" is supported at this time, with additional event-type-actions being added later when required.